src/style.scss

```diff
@@ -0,0 +1,9 @@
+@import url("https://fonts/.googleapis.com/css?family=Karla:weight@400;700&display=swap");
+
+$font: "Karla", sans-serif;
+$primary-color: #3e6f9e;
+
+body {
+  font-family: $font;
+  color: $primary-color;
+}
```

Comment:
[OPTIONAL] In order to make your life easier, avoid code repetition, and auto-generate cleaner code, consider adding your CSS code in the '.scss' file and let the 'watch SASS'  (the vscode addon) do the hard work for you. It will update your '.css' automatically everytime you update the '.scss' file.

Reply:
Thank you for that suggestion, tried it and it worked !. Awesome!. @mrjuda